Sources/NIOHTTP1Server/main.swift

```diff
@@ -424,7 +432,12 @@ private final class HTTPHandler: ChannelInboundHandler {
 }
 
 // First argument is the program path
-let arguments = CommandLine.arguments
+var arguments = CommandLine.arguments.dropFirst(0)
```

Comment:
Presumably this is an attempt to coerce the type?

Reply:
correct :)

Reply:
Mind writing that in a comment just so others can see why that was done?